modules/gst/gst.c

```diff
@@ -13,6 +13,7 @@
 #include <re.h>
 #include <rem.h>
 #include <baresip.h>
+#include <unistd.h>
```

Comment:
this is an unrelated change, please make a separate PR for it.
stating the problem would also be good, i.e. why the change is needed